### DISCUSSION TITLE
CSS syntax error in tag 'style amp-custom'. error

### DISCUSSION THREAD

**Discussion by uprootpc** [Automattic/amp-wp#6386] [Q&A]
Hi, search console returned this warning CSS syntax error in tag 'style amp-custom'. for one my pages. kindly advise how to rectify it

**Comment by milindmore22** [comment]
Hello @uprootpc 

Please create a support topic on the WordPress support forum https://wordpress.org/support/plugin/amp/#new-topic-0

**Comment by westonruter** [comment]
You'll probably want to refer to https://github.com/ampproject/amphtml/pull/34553 as well. The validator change has been pushed out which should have fixed this, but you can see from https://github.com/ampproject/amphtml/pull/34553#issuecomment-857253324 that warnings are still happening.
